### DISCUSSION TITLE
`short-schedule:run` not working

### DISCUSSION THREAD

**Discussion by n1c0j0n3s** [spatie/laravel-short-schedule#23] [Q&A]
Hello.

I have been trying to setup a job that runs every second with no luck in a Dockerized Laravel 8 application with PHP 7.4.

After installing the package via composer, I added this to the Kernel.php file:

```php
protected function shortSchedule(ShortSchedule $shortSchedule)
{
    $shortSchedule->command('save-ticker')->everySecond();
}
```

This is my Command definition:

```php
class SaveTickerCommand extends Command
{
    
    protected $signature = 'save-ticker:run';
    protected $description = 'Populate DB with ticker data';

    public function __construct()
    {
        parent::__construct();
    }

    public function handle(TickerServiceProvider $tickerServiceProvider)
    {
        $tickerServiceProvider->saveTickers();
        info("Command run");
    }
}
```

The `handle` method receives a `TickerServiceProvider` object. I removed the service injection to test and still does not work.

If I run `php artisan save-ticker:run` it works fine. But `php artisan short-schedule:run` does nothing. The prompt stays in standby, but nothing happens.

Any suggestion?

**Comment by rodrigorosa1** [comment]
I have same problem... You solved?

**Comment by nikolacolic93** [comment]
Hi,

Since your command's signature is `'save-ticker:run'` I think your `shortSchedule` method should look like this:

```
protected function shortSchedule(ShortSchedule $shortSchedule)
{
    $shortSchedule->command('save-ticker:run')->everySecond();
}
```
